neuralmonkey/config/utils.py

```diff
@@ -30,13 +33,13 @@ def dep_func(*args, **kwargs):
 initialize_vocabulary = vocabulary.initialize_vocabulary
 
 
-def adam_optimizer(learning_rate=1e-4):
+def adam_optimizer(learning_rate=1e-4) -> tf.train.AdamOptimizer:
     return tf.train.AdamOptimizer(learning_rate)
 
 
-def adadelta_optimizer(**kwargs):
+def adadelta_optimizer(**kwargs) -> tf.train.AdadeltaOptimizer:
     return tf.train.AdadeltaOptimizer(**kwargs)
 
 
-def variable(initial_value=0, trainable=False, **kwargs):
+def variable(initial_value=0, trainable=False, **kwargs) -> tf.Variable:
```

Comment:
tak tam dej Union[int, tf.Tensor] - jestli někdo někdy použije něco jinýho, tak to buď přidá do toho sjednocení, nebo si to přepíše. A bool je bool, ne?